src/main/java/com/pio/studentManagement/entity/Student.java

```diff
@@ -0,0 +1,75 @@
+package com.pio.studentManagement.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+
+@Entity
+@Table(uniqueConstraints = {@UniqueConstraint(name = "unique_contact", columnNames = "studentContact"), @UniqueConstraint(name = "unique_email", columnNames = "studentEmail")})
+public class Student {
+    @Id()
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int studentId;
```

Comment:
```suggestion
    private int id;
```
remove prefix student before each student property name